operator/cilium_node.go

```diff
@@ -40,7 +48,8 @@ func startSynchronizingCiliumNodes(apiextensionsK8sClient apiextensionsclientset
 	// TODO: The operator is currently storing a full copy of the
 	// CiliumNode resource, as the resource grows, we may want to consider
 	// introducing a slim version of it.
-	_, ciliumNodeInformer := informer.NewInformer(
+	var ciliumNodeInformer cache.Controller
+	CiliumNodeStore, ciliumNodeInformer = informer.NewInformer(
```

Comment:
Given the previous suggested change:
```suggestion
	ciliumNodeStore, ciliumNodeInformer = informer.NewInformer(
```

Reply:
Done.